loggingservice/logging.yaml

```
@@ -69,6 +69,19 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
+
+  /logs/version:
+    get:
+      summary: Get version of logging building block
+      tags:
+        - Logs
+      responses:
+        200:
+          description: The version of logging building block.
```

```suggestion
          description: The version of logging building block.
          content:
            text/plain:
              schema:
                type: string
```

Please use plain text response type for this endpoint. This is because other endpoints are using the same response type. Please also use the same in the Contributions BB version endpoint. Thanks.

---

Changed

---

Thanks.